src/CodeGenerator/InjectorGenerator.php

```diff
@@ -57,48 +55,40 @@ class InjectorGenerator
      */
     protected $definition;
 
-    /**
-     * @var string
-     */
+    /** @var string */
     private $namespace;
 
-    /**
-     * @var FactoryGenerator
-     */
+    /** @var FactoryGenerator */
     private $factoryGenerator;
 
-    /**
-     * @var AutoloadGenerator
-     */
+    /** @var AutoloadGenerator */
     private $autoloadGenerator;
 
-    /**
-     * @var LoggerInterface
-     */
+    /** @var LoggerInterface */
     private $logger;
 
     /**
      * Constructs the compiler instance
      *
-     * @param ConfigInterface $config The configuration to compile from
-     * @param DependencyResolverInterface $resolver The resolver to utilize
-     * @param string|null $namespace Namespace to use for generated class; defaults
-     *     to Zend\Di\Generated.
-     * @param LoggerInterface|null $logger An optional logger instance to log failures
-     *     and processed classes.
+     * @param ConfigInterface             $config    The configuration to compile from
+     * @param DependencyResolverInterface $resolver  The resolver to utilize
+     * @param string|null                 $namespace Namespace to use for generated class; defaults
+     *                     to Zend\Di\Generated.
```

Comment:
@webimpress  The autofixer for the alignment rule does some odd things here? What is this indentation based upon?